frameworks/preact.md

```diff
@@ -136,6 +136,10 @@ and its corresponding `ReloadPrompt.css` styles file:
 ```
 :::
 
+::: warning
+The options provided to hooks are not reactive. Therefore, the callback references will be the first rendered options instead of the latest hook’s options. If you are doing complex logic with state changes, you will need to provide a stable reference function.
```

Comment:
Can you create `ReactReactiveWarning.md` inside vitepress components fokder and replace both warnings with the component?

Move it at top, below the heading section.

Reply:
Done
![image](https://github.com/vite-pwa/docs/assets/18393696/006cf19a-a20e-4a66-a1c9-5b2d4ebce187)